core/trace/src/main/scala/org/typelevel/otel4s/trace/Tracer.scala

```diff
@@ -204,4 +204,6 @@ object Tracer {
       def spanBuilder(name: String): SpanBuilder.Aux[F, Span[F]] = builder
       def joinOrRoot[A, C: TextMapGetter](carrier: C)(fa: F[A]): F[A] = fa
     }
+
+  implicit def noop[F[_]: MonadCancelThrow]: Tracer[F] = noop
```

Comment:
this should be inside an `Implicits` object

```suggestion
  object Implicits {
    implicit def noop[F[_]: MonadCancelThrow]: Tracer[F] = Tracer.noop
  }
```